content/en/admin/tootctl.md

```diff
@@ -641,14 +641,23 @@ Fix corrupted database indexes that may have been caused due to changing collati
 
 ### `tootctl media remove` {#media-remove}
 
-Remove locally cached copies of media attachments from other servers.
+Removes locally cached copies of media attachments, avatars or profile headers from other servers. By default, only media attachments are removed.
 
 `--days N`
 : How old media attachments have to be before they are removed. Defaults to 7.
 
 `--concurrency N`
 : The number of workers to use for this task. Defaults to N=5.
 
+`--prune-profiles`
+: Only removes avatars and headers.
```

Comment:
```suggestion
: Instead of media attachments, remove locally cached copies of avatars and headers from other servers. Cannot be combined with --remove-headers.
```